### ISSUE TITLE
dpkg-buildpackage try to build a source package after I passing --only binary to debspawn

### ISSUE THREAD

**Issue by weilinfox** [lkhq/debspawn#29]
When I try to build packages with ``debspawn`` in source directory, I passed ``--only binary`` option to build binary package only, but it still try to invoke ``dpkg-buildpackage -S`` to build a source package.

Is there any way to build a binary package without a source package using ``debspawn`` ?

My command below:

```bash
$ ./debspawn.py --verbose build --only binary jammy .
```

Thanks in advance :-)

**Comment by ximion** [comment]
Can you describe what exactly you are trying to do? The `--only binary` option will result in only the binary packages being built & exported in a changes file in the end, but to actually generate those, debspawn needs a temporary source to build from, hence a source package is generated temporarily as normal part of the build process when building from a source tree, to ensure the resulting packages are sane.

**Comment by weilinfox** [comment]
I used to use ``mk-build-deps`` and ``dpkg-buildpackage -b --sign-key=xxx`` to generate a binary package without a source package. You see, ``mk-build-deps`` will install build dependencies into my host system, so I'm trying to find a tool to build them in a container. 

What I have tried to do is build the package in a source directory which directly cloned from github repo, with ``debian/`` in it, just like the image attached below. So when ``dpkg-buildpackage`` try to build a source package, it failed to find a upstream tarball.

![image](https://user-images.githubusercontent.com/54651938/173271100-6f5ab52d-1e77-4f63-b389-df69d3a2beda.png)

I have read more code yesterday, it seems I have misunderstood the meaning of ``--only binary`` option.   >_<
